### DISCUSSION TITLE
github copliot  chat can not response in goland but works well in vscode

### DISCUSSION THREAD

**Discussion by yiyanwannian** [community/community#120507] [Copilot Conversations]
### Select Topic Area

Bug

### Body

my goland github copliot plugin can not response in chat, I have paied this  services for an year, but this chat feature works well on my vscode

**Comment by yiyanwannian** [comment]
![image](https://github.com/community/community/assets/8225547/5af17305-08c8-4b86-bb9a-52de8f0c66e9)

**Reply by achenchenxx** [reply]
IDEA 也长时间无反应

**Reply by laodaocheney** [reply]
always "Responding..." in Goland 2024.1, plugin version is: 1.5.2.5345

**Comment by xiaoshi1989** [comment]
hasn't it been fixed yet?

**Reply by xiaoshi1989** [reply]
idea 1.5.2.5345

**Reply by BeSuccessfulMan** [reply]
no sitll broken

**Comment by BeSuccessfulMan** [comment]
me too! 2024.4.26

**Comment by BeSuccessfulMan** [comment]
![image](https://github.com/community/community/assets/74354036/b038ab9e-6aad-4252-b902-a0086e555525)
always responding

**Reply by BeSuccessfulMan** [reply]
hour and hour and hour...

**Comment by yuxiuliang** [comment]
![微信截图_20240508110628](https://github.com/community/community/assets/10753309/e5361aa7-d04f-4f8f-b19c-0b0e21c0d6f1)
Unable to use the chat function in IntelliJ IDEA.

**Comment by enzo23333** [comment]
me too
